diff -up efl-1.21.0/src/lib/efl/interfaces/efl_ui_format.c.pragma efl-1.21.0/src/lib/efl/interfaces/efl_ui_format.c
--- efl-1.21.0/src/lib/efl/interfaces/efl_ui_format.c.pragma	2018-08-18 15:27:24.073708072 -0400
+++ efl-1.21.0/src/lib/efl/interfaces/efl_ui_format.c	2018-08-18 15:28:38.690980709 -0400
@@ -130,7 +130,10 @@ _default_format_cb(void *data, Eina_Strb
      }
    else if (sd->format_type == FORMAT_TYPE_STATIC)
      {
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wformat-security"
         eina_strbuf_append_printf(str, sd->template);
+#pragma GCC diagnostic pop
      }
    else
      {
